lib/code_fund_web/controllers/ad_serve_controller.ex

```diff
@@ -86,6 +87,11 @@ defmodule CodeFundWeb.AdServeController do
         )
         |> details_render(conn)
 
+      true ->
+        conn
+        |> error_details(property_id, "Your country cannot view sponsorships at this time")
```

Comment:
Let's replace this text with "CodeFund does not have an advertiser for you at this time"

Reply:
sounds good, i figured that would change